modules/webgl/src/webgl-utils/webgl-types.js

```diff
@@ -18,7 +18,12 @@ typical configuration for isorender applications running on the server.`;
 // TODO(Tarek): OOGLY HACK to avoid webpack requiring headless
 //   browser bundles. Will be removed in 8.0 when we
 //   remove automatic headless context creation
-const m = module;
+let m;
```

Comment:
Let's add a note that this try/catch fixes rollup support to help future webpack-using maintainers.

Reply:
Added.
```
// NOTE: Rollup does not process the line `const m = module;`
//   and writes it out verbatim in its final output, which ends
//   up falling over in browser environments at runtime. Added
//   a `try/catch` block to fix usage in rollup builds.
```